index.js

```diff
@@ -0,0 +1,65 @@
+
+let playerObj = "http://localhost:3000/players/1"
+
+document.addEventListener('DOMContentLoaded', function() {
+    fetchPlayer()
+})
+
+function fetchPlayer() {
+    return fetch(playerObj)
+    .then(res => res.json())
+    .then(player => renderPlayer(player))
+}
+
+function renderPlayer(player) {
+    const img = document.querySelector(".player img")
+    img.src = player.photo
+    img.alt = player.name
+
+    const name = document.querySelector(".player h2")
+    name.textContent = player.name
+
+    const nickname = document.querySelector(".player em")
+    nickname.textContent = player.nickname
+
+    const likes = document.querySelector(".player .likes")
+    likes.textContent = `${player.likes} Likes`
+
+    player.goals.forEach(renderGoal)
+}
+
+function fetchLikes() {
+    return fetch(playerObj)
+    .then(res => res.json())
+    .then(player => addLike(player))
+}
+
+const heartBtn = document.querySelector(".like-button")
+
+heartBtn.addEventListener("click", addLike)
+
+function addLike(player) {
+    player.likes += 1
+}
```

Comment:
This event listener won't work - `player` isn't going to be passed into the function. You'd need to access the player's likes from the DOM in order to update the number, and also need to write the code for the `fetch` to persist on the server.